CHANGELOG

```diff
@@ -2,6 +2,9 @@ libseccomp-golang: Releases
 ===============================================================================
 https://github.com/seccomp/libseccomp-golang
 
+* Version UNRELEASED - FUTURE
```

Comment:
I appreciate the gesture, but please refrain from adding things to the CHANGELOG, @drakenclimber and I update that as part of the release process.

Reply:
ok, I will remove that change.